ogre/include/ignition/rendering/ogre/OgreLidarVisual.hh

```diff
@@ -60,6 +60,11 @@ namespace ignition
       public: virtual void SetPoints(
               const std::vector<double> &_points) override;
 
+      // Documentation inherited
```

Comment:
can you add a note here that it currently only affects lidar visuals with type `LVT_POINTS`?